internal/expr/expr_test.go

```diff
@@ -50,6 +50,12 @@ var tests = []struct {
 	prepareArgs      []any
 	expectedPrepared string
 }{{
+	"function with nested input",
+	`SELECT max($Person.id, 20) AS &Manager.id FROM person`,
+	`[Bypass[SELECT ] Output[[[Bypass[max(] Input[Person.id] Bypass[, 20)]]] [Manager.id]] Bypass[ FROM person]]`,
+	[]any{Person{}, Manager{}},
+	`SELECT max(@sqlair_0, 20) AS _sqlair_0 FROM person`,
+}, {
```

Comment:
Would it be good to add a test where the function calls are nested? E.g.
```sql
SELECT max(min(100, $Person.id), 20) AS &...
```

Reply:
I do have a test for this, the only reason this test was separate (at the top) was so I could debug it more easily. I have added a second test for having multiple input statements in different levels of the the nested functions though.